llvm-15.0.3/llvm-crash-analyzer/lib/Analysis/ConcreteReverseExec.cpp

```diff
@@ -130,9 +130,50 @@ template <typename T> std::string intToHex(T num, unsigned regValSize) {
   return stream.str();
 }
 
+void ConcreteReverseExec::updatePC(const MachineInstr &MI) {
+  // If the option is enabled, we skip the CRE of the MIs.
+  if (DisableCRE || !getIsCREEnabled())
+    return;
+  // Initial PC value for the frame, points to the crash-start instruction.
+  // We start updating PC for instructions preceding to the crash-start.
+  if (MI.getFlag(MachineInstr::CrashStart))
+    return;
+
+  auto CATI = getCATargetInfo();
+  if (!CATI->getPC())
+    return;
+  std::string RegName = *CATI->getPC();
+
+  auto regVal = getCurretValueInReg(RegName);
```

Comment:
That's a nice catch! I will fix it in https://github.com/cisco-open/llvm-crash-analyzer/pull/41.
I wanted to keep instruction size&address map only for analyzed instructions (before crash-start), because of the performance concerns.